installer/pyinstaller/build-linux.sh

```diff
@@ -16,7 +16,7 @@ if [ "$python_version" = "" ]; then
 fi
 
 if [ "$openssl_version" = "" ]; then
-    openssl_version="3.0.15";
+    openssl_version="3.3.1";
```

Comment:
Should we also make sure there's no CVE's detected in the new version being used? We can also monitor that once this PR is merged.

Reply:
Good point, it seems like there is no report about this version yet (only some fixes). https://openssl-library.org/news/vulnerabilities/index.html

I agree that we should monitor pipeline afterwards